### PR TITLE
📝 Add team ceremonies to documentation

### PR DESCRIPTION
**Pull request by bagg3rs**
This PR adds a page to capture how we as a team run our ceremonies

image

### CODE REVIEW

File: docs/source/documentation/team/our-ceremonies.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    Starting with the "Done" column we discuss any active issues in GitHub via [ZenHub](https://app.zenhub.com/workspaces/data-platform-core-infrastructure-63e3ab98f777b90011c85bbb/board)
+
+- **Yesterday, Today, Blocked**
+
+    A round robin of each team member discussing their issues and any blockers.
+
+## Sprint Planning 
+
+Each Sprint is two weeks long and starts on a Thursday.
+We use Miro or Jamboard to run a one-hour session. Template agenda below:
```
Comment (YvanMOJdigital) [inline docs/source/documentation/team/our-ceremonies.html.md.erb]: We just use the zenhub board for planning

File: docs/source/documentation/team/our-ceremonies.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+1. Reveal all stickies and vote for your favourite 5 votes (2m).
+1. Discuss and finalise any concerns (5m).
+1. Pull issues into the sprint.
+1. Any other business.
+
+## Backlog Refinement
+
+On Thursday afternoons we have a forty-five minute session on backlog refinement. The purpose of this ceremony is for the team to agree that any issues created have enough information for any other team member to pick up.  Once a issue is refined we estimate it using the [Fibonacci scale](https://en.wikipedia.org/wiki/Fibonacci_scale_(agile)).
+
+## Backlog Prioritisation
```
Comment (YvanMOJdigital) [inline docs/source/documentation/team/our-ceremonies.html.md.erb]: We doing this?

Comment (bagg3rs) [reply docs/source/documentation/team/our-ceremonies.html.md.erb]: No, but do we want to?

File: docs/source/documentation/team/our-ceremonies.html.md.erb (hunk 1 above)
Comment (bagg3rs) [reply docs/source/documentation/team/our-ceremonies.html.md.erb]: updated

Comment (YvanMOJdigital) [reply docs/source/documentation/team/our-ceremonies.html.md.erb]: For sprint planning: 

1. Discuss the previous sprint and if we met our objective/estimated tickets, did we achieve anything outside of our goal? 
2. Review any tickets caring over to the next sprint and why
3. Agree team capacity for the sprint
4. Pull any new issues into the sprint backlog and ensure all tickets have sufficient descriptions, epics assigned and estimated size
5. AOB

**Review by jacobwoffenden — APPROVED** [review]

**Pull request merged**